core/src/string/avm_string.rs

```diff
@@ -38,6 +38,13 @@ impl<'gc> AvmString<'gc> {
         }
     }
 
+    pub fn as_managed(self) -> Option<Gc<'gc, AvmStringRepr<'gc>>> {
```

Comment:
Do we use the 'managed' terminology anywhere else? If not, could we call this `as_owned() `?

Reply:
Oh sorry, forgot to mention (I also intended to add a comment).
Ideally the `Owned` naming is extremely misleading, since a `Source::Owned` string can either be dependent ("owned") or non-dependent ("owner"). IMO to resolve the overloaded usage of the term, the `Source::Owned` should become `Source::Managed` (in the "gc-managed" sense - feel free to propose alternatives), and owned/owner terminology will only relate to dependent strings.

I didn't want to refactor this naming in this PR, but I did "pave the way" with `.as_managed`, since otherwise having `left.owner().as_owned()` sounds like utter nonsense.